src/win/tty.c

```diff
@@ -2331,6 +2352,20 @@ static void CALLBACK uv__tty_console_resize_event(HWINEVENTHOOK hWinEventHook,
                                                   LONG idChild,
                                                   DWORD dwEventThread,
                                                   DWORD dwmsEventTime) {
+  SetEvent(uv__tty_console_resized);
```

Comment:
`uv__tty_console_resized` can be NULL here, I think? Is that bad?

Reply:
It cannot, it wont start the watcher if the event could not be created.